src/Compilers/Core/Portable/ConstantValueSpecialized.cs

```diff
@@ -201,6 +210,21 @@ internal override string GetValueToDisplay()
             {
                 return (_value == null) ? "null" : string.Format("\"{0}\"", _value);
             }
+
+            public override string ToString(string? format, IFormatProvider? provider)
```

Comment:
>ToString [](http://example.com/codeflow?start=35&length=8)

Consider adding a test (to `ConstantTests`) for various format strings and lengths.